submissions/yemilyz.md

```diff
@@ -0,0 +1,10 @@
+* submitter: @yemilyz
+* statement: I hereby agree that the facts I have submitted are my own, and that I grant a license for Pryaxis Limited, or designated entities by Pryaxis Limited, the ability to use this data in a way to promote future Pryaxis services.
+
+----
+
+* @American Eskimos are mistrustful, so they are not great around strangers and will tend to bark a lot.
+* Their personality and intelligence made them ideal for circus performances back in the day (early 20th century).
```

Comment:
This line is neither marked with a subject signifier (```@American Eskimos```) nor is it in a list. Can you add a fact signifier somewhere in that sentence?